### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.5 - es. 3.26 (21 ottobre)

### DISCUSSION THREAD

**Discussion by alem1105** [sapienzastudentsnetwork/calcolo-delle-probabilita#300] [Esercizi M-Z]
<img width="965" alt="immagine" src="https://github.com/user-attachments/assets/08f79c9c-7464-4549-a6d8-be7ef713ce83">

**Comment by alem1105** [comment]
![image](https://github.com/user-attachments/assets/b6bdcd18-2c30-4f88-8eea-a4a635a004ea)

**Comment by AglaiaNorza** [comment]
![image](https://github.com/user-attachments/assets/74c2ecea-a6a2-42c7-80e2-1a78c64ccb8b)

**Comment by marcoseverini** [comment]
![immagine](https://github.com/user-attachments/assets/080f9685-db42-4068-a777-daa438a21be7)
